apps/molgenis-components/src/queryEmx2/queryEmx2.ts

```diff
@@ -3,29 +3,32 @@
  */
 
 import { request } from "graphql-request";
+import { IColumn, ITableMetaData } from "../../../meta-data-utils/dist";
 
 class QueryEMX2 {
   tableId = "";
-  filters = {};
+  filters: Record<string, any> = {};
   column = "";
-  _schemaTablesInformation = {};
+  _schemaTablesInformation: Record<string, any> = {};
   selection = ["id", "name"];
   graphqlUrl = "";
   branch = "root";
-  limits = {};
-  orderings = {};
+  limits: Record<string, number> = {};
+  orderings: Record<string, { column: string; direction: "asc" | "dsc" }> = {};
   findInAllColumns = "";
-  page = {};
+  page: Record<string, any> = {};
   aggregateQuery = false;
+  type = "";
+  orCount = 0;
```

Comment:
note: these 2 variables where not initialized originally, which was a bug (though I don't know the symptoms).